### PR TITLE
[MOBILE-335] Stop tracking entire Airship folder

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Adds a script that downloads and installs the Airship folder using the npm postinstall hook

### Why are these changes necessary?
To no longer track the entire Airship folder - and to instead have the Airship folder contents always match the static lib version defined in the plugin.xml

### How did you verify these changes?
Created and ran the sample

### CODE REVIEW

**Comment by crow** [comment]
Going to re-write the script in JS, for better cross-platform compatibility

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "minami": "1.2.3"
   },
   "scripts": {
-    "generate-docs": "jsdoc ./www/UrbanAirship.js -t ./node_modules/minami/ -r jsdoc_readme.md"
+    "generate-docs": "jsdoc ./www/UrbanAirship.js -t ./node_modules/minami/ -r jsdoc_readme.md",
+    "postinstall" : "source scripts/update_ios_static_lib.sh"
```
Comment (rlepinski) [inline package.json]: If you do this in a publish step instead, we can use whatever we want.

**Comment by rlepinski** [comment]
👍

File: scripts/update_ios_static_lib.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Parse framework version line from plugin and pull out the version number
+VERSION=$(xpath plugin.xml '/plugin/platform[@name="ios"]/source-file[@framework="true"]' 2>/dev/null |
+grep -oh '[0-9]\+\.[0-9]\+\.[0-9]\+')
+
+echo "Downloading libUAirship-$VERSION.zip from bintray..."
+curl -s -LO "https://urbanairship.bintray.com/iOS/urbanairship-sdk/$VERSION/libUAirship-$VERSION.zip"
```
[the remark this answers is not in the document]
Comment (crow) [reply scripts/update_ios_static_lib.sh]: I'm gonna stick with keeping this outside the temp file, the temp file's purpose is to hold the multiple files from the unzip process

[the remark this answers is not in the document]
Comment (BrianBatchelder) [reply scripts/update_ios_static_lib.sh]: As long as you are confident of the contents of the directory you are using, it's OK.

**Pull request merged**